daml_dit_if/main/config.py

```diff
@@ -44,7 +45,7 @@ def envint(var: str, default: 'Optional[int]' = None) -> int:
         return int(val)
     except ValueError:
         FAIL(f'Invalid integer {val} in environment variable: {var}')
-
+        return -1  # unreached
```

Comment:
If you give `FAIL` a return of `typing.NoReturn`, you won't need this.

Reply:
Nice feature. Resovled.